ChatQnA/benchmark_chatqna.yaml

```diff
@@ -28,10 +28,16 @@ deploy:
       memory_capacity: ""
 
     llm:
-      engine: tgi
-      model_id: ""
-      replicaCount: [7, 15, 31, 63]
+      engine: vllm  # or tgi
+      model_id: "meta-llama/Meta-Llama-3-8B-Instruct"
+      replicaCount:
+        with_teirerank: [7, 15, 31, 63]     # When teirerank.enabled is True
+        without_teirerank: [8, 16, 32, 64]   # When teirerank.enabled is False
```

Comment:
would it be better to specify as with_vllm? tei as a binary works provided only two server options.

Reply:
But this parameter is common used for tgi and vllm engine.